opennms-base-assembly/src/main/filtered/etc/magic-users.properties

```diff
@@ -73,3 +73,7 @@ role.rest.users=iphone
 
 role.mobile.name=OpenNMS Mobile User
 role.mobile.users=
+
+role.minion.name=OpenNMS Minion User
```

Comment:
Considering that https://github.com/OpenNMS/opennms/pull/1009 has been waiting for a while (and it is pretty much ready to be merged), guess we should decide if this one will be merged first, and then I have to rework the solution for HZN-871, or merge 1009 first, and then use the new methodology here (which is a lot simpler and easy to use).